build/scons-xcode-project-build.js

```diff
@@ -13,7 +13,12 @@ const projectDir = program.args[0];
 const targetBuildDir = program.args[1];
 const productName = program.args[2];
 
-const appDir = path.join(targetBuildDir, `${productName}.app`);
+const ROOT_DIR = path.join(__dirname, '..');
```

Comment:
<!--
  0 failure: 
  1 warning:  build/scons-xcode...
  
  
  DangerID: danger-id-Danger;
  File: build/scons-xcode-project-build.js;
  Line: 16;
-->

- :warning: build/scons-xcode-project-build.js line 16 – 'ROOT_DIR' is assigned a value but never used. (no-unused-vars)